deploy/template.yaml

```diff
@@ -359,6 +359,34 @@ Resources:
         Owner: !Ref OwnerTagValue
         Source: !Ref SourceTagValue
 
+  APIGatewayIDParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Description: SSM parameter for API Gateway ID reference
+      Name: !Sub "/${AWS::StackName}/ApiGateway/ID"
+      Type: String
+      Value: !Ref ApiGateway
+      Tags:
+        Product: !Ref ProductTagValue
+        System: !Ref SystemTagValue
+        Environment: !Ref Environment
+        Owner: !Ref OwnerTagValue
+        Source: !Ref SourceTagValue
+
+  ECSServiceNameParameter:
```

Comment:
And could we put this resource just beneath the ContainerService resource please